src/Common/Update.php

```diff
@@ -119,12 +119,13 @@ public function orWhere($cond)
      *
      * @param string $col The column name.
      *
-     * @return $this
+     * @param array  $value
```

Comment:
i looked at the code and some `@params` had a blank line between, some did not. What is your preference? I'll fix the others too...